Watership_Down_Exercise/Program.cs

```diff
@@ -8,12 +8,14 @@ namespace Watership_Down_Exercise
 {
     class Program
     {
+        //The main program
         static void Main(string[] args)
         {
+            const int NUMBER_OF_BUNNIES = 5;
             List<Bunny> bunnies = new List<Bunny>();
-            /* Create 5 bunnies */
-           
-            for (int i = 0; i < 5; i++)
+
+            /* Create the bunnies */
+            for (int i = 0; i < NUMBER_OF_BUNNIES; i++)
             {
                 Bunny newBunny = BunnyCreator.CreateBunny();
                 bunnies.Add(newBunny);
```

Comment:
I think it's out of date according to the hill class you created.
It should only run the hill